inc/template-tags.php

```diff
@@ -391,6 +391,56 @@ function responsive_utility_nav( $args = array() ) {
 	}
 }
 
+/**
+ * Renders short navigation menu.
+ *
+ * If the current site has a site-wide ACL applied or the short menu has
+ * no items nothing will be displayed.
+ *
+ * @param array $args {
+ *     Optional. Arguments to configure menu markup.
+ *
+ *     @type  string $before HTML markup to display before menu.
+ *     @type  string $after  HTML markup to display after menu.
+ * }
+ */
+function responsive_short_nav( $args = array() ) {
+	if ( ! has_nav_menu( 'short' ) ) {
+		return;
+	}
+
+	$after  = '<div class      = "menu-bar">';
+	$after .= '<button type    = "button" class                                                                                                                      = "nav-toggle js-nav-toggle" aria-label = "' . __( 'Open menu', 'responsive-framework' ) . '" aria-expanded = "true">';
+	$after .= '<div class      = "nav-toggle-label-closed">' . apply_filters( 'responsive_mega_menu_closed', __( 'Full Menu', 'responsive-framework' ) ) . '</div>';
+	$after .= '<div class      = "nav-toggle-label-open">' . apply_filters( 'responsive_mega_menu_opened', __( 'Close Menu', 'responsive-framework' ) ) . '</div>';
+	$after .= '<span></span>';
+	$after .= '</button>';
+	$after .= '</div>';
```

Comment:
This entire block is a little hard to read, and also because we're writing HTML in a string we shouldn't have spaces between equals signs and quotation marks (I think you might have been thinking you needed to line them up for phpcs).

I would remove the extra spaces between all the quotation and spaces for starters.

Personally, when I write large chunks of HTML to a variable, I try to do it like this:

```
$after = '
	<div class="menu-bar">
		<button type="button" class="nav-toggle js-nav-toggle" aria-label="' . __( 'Open menu', 'responsive-framework' ) . '" aria-expanded="true">
			<div class="nav-toggle-label-closed">' . apply_filters( 'responsive_mega_menu_closed', __( 'Full Menu', 'responsive-framework' ) ) . '</div>
			<div class="nav-toggle-label-open">' . apply_filters( 'responsive_mega_menu_opened', __( 'Close Menu', 'responsive-framework' ) ) . '</div>
			<span></span>
		</button>
	</div>
';
```

Which looks a little more legible at first glance